dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -5816,13 +5786,13 @@ public void copyProperties(final Contentlet contentlet, final Map<String, Object
                             contentlet.setProperty(conVariable, value);
                         }
                     }else if(isFieldTypeBoolean(field)){
-                        contentlet.setBoolProperty(conVariable, value != null ? (Boolean)value : null);
+                        contentlet.setBoolProperty(conVariable, value != null ? (Boolean)value : false);
```

Comment:
I would use ConversionUtils.toBooleanFromDb

Reply:
and ConversionUtils has another methods to convert objects, take a look

Reply:
Done